ui/components/ui/tabs/tabs.component.js

```diff
@@ -1,19 +1,22 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import Box from '../box';
+import { Box } from '../../component-library';
 import {
   BackgroundColor,
-  DISPLAY,
+  Display,
   JustifyContent,
 } from '../../../helpers/constants/design-system';
 
 const Tabs = ({
-  defaultActiveTabKey,
-  onTabClick,
+  defaultActiveTabKey = '',
+  onTabClick = () => null,
   children,
-  tabsClassName,
-  subHeader,
+  tabsClassName = '',
+  subHeader = null,
+  tabListProps = {},
+  tabContentProps = {},
```

Comment:
Opting to set prop defaults instead of passing falsey props at each instance of the `Tabs` component.